tasks1(difference)/Program1.cs

```diff
@@ -0,0 +1,14 @@
+﻿    Dictionary<Tvalue1, Tvalue2> Difference<Tvalue1, Tvalue2>
+        (Dictionary<Tvalue1, Tvalue2> dict1, Dictionary<Tvalue1, Tvalue2> dict2)
+    {
+        foreach (Tvalue1 dictKey in dict2.Keys)
+            dict1.Remove(dictKey);

```

Comment:
Don't change incoming parameters

Reply:
Are you talking about type of parameters or parameters names?If about type then i dont know on which type i must replace dictionary type, cause dictionary replaces js objects the best in this case.